### PR TITLE
docs(cn): api/ssr.md translation

### PR DESCRIPTION
**Pull request by pakchoily**
## Additional Information
pr #219 
finish `ssr.md` translation task

### CODE REVIEW

**Comment by Jinjiang** [comment]
Will hold this PR on as a draft status for a wait until #219 merged. Thanks.

**Comment by Jinjiang** [comment]
@pakchoily I found there are some conflicts need to be solved. Would you mind taking a look? Thanks.

**Comment by pakchoily** [comment]
> @pakchoily I found there are some conflicts need to be solved. Would you mind taking a look? Thanks.

The conflict has been resolved.

File: src/api/ssr.md [hunk 1]
```diff
@@ -30,11 +30,11 @@
   })()
   ```
 
-  ### Handling Teleports
+  ### 处理 Teleport
 
-  If the rendered app contains Teleports, the teleported content will not be part of the rendered string. In most cases, the best solution is to conditionally render the Teleport on mount.
+  如果渲染的应用包含 Teleport，那么 teleport 内容将不会作为渲染字符的一部分。在大多数情况下，最佳方案是在挂载时有条件的渲染 Teleport。
```
Comment (Jinjiang) [inline src/api/ssr.md:35]:
```suggestion
  如果渲染的应用包含 Teleport，那么 teleport 的内容将不会作为渲染字符串的一部分。在大多数情况下，最佳方案是在挂载时条件式地渲染 Teleport。
```

File: src/api/ssr.md [hunk 2]
```diff
@@ -30,11 +30,11 @@
...
 
-  If you do need to hydrate teleported content, they are exposed under the `teleports` property of the ssr context object:
+  如果你需要激活 teleport 内容，他位于服务端渲染上下文对象的 `teleports` property中：
```
Comment (Jinjiang) [inline src/api/ssr.md:37]:
```suggestion
  如果你需要激活 teleport 内容，服务端渲染上下文对象将它们暴露在了 `teleports` property 下：
```

File: src/api/ssr.md [hunk 3]
```diff
@@ -47,9 +47,9 @@
```
Comment (Jinjiang) [inline src/api/ssr.md:47]: L46 待翻译

File: src/api/ssr.md [hunk 4]
```diff
@@ -47,9 +47,9 @@
 
 ## renderToNodeStream()
 
-Renders input as a [Node.js Readable stream](https://nodejs.org/api/stream.html#stream_class_stream_readable).
+渲染输入作为 [Node.js Readable stream](https://nodejs.org/api/stream.html#stream_class_stream_readable)。
```
Comment (Jinjiang) [inline src/api/ssr.md:50]:
```suggestion
[Node.js Readable stream](https://nodejs.org/api/stream.html#stream_class_stream_readable) 形式的渲染输入。
```

File: src/api/ssr.md [hunk 5]
```diff
@@ -63,19 +63,19 @@ Renders input as a [Node.js Readable stream](https://nodejs.org/api/stream.html#
 - **示例**
 
   ```js
-  // inside a Node.js http handler
+  // 内置 Node.js http 处理器
```
Comment (Jinjiang) [inline src/api/ssr.md:66]:
```suggestion
  // 在一个 Node.js http 处理函数内
```

File: src/api/ssr.md [hunk 6]
```diff
@@ -63,19 +63,19 @@ Renders input as a [Node.js Readable stream](https://nodejs.org/api/stream.html#
...
   renderToNodeStream(app).pipe(res)
   ```
 
-  :::tip Note
-  This method is not supported in the ESM build of `vue/server-renderer`, which is decoupled from Node.js environments. Use [`pipeToNodeWritable`](#pipetonodewritable) instead.
+  :::tip 备注
+  ESM构建的 `vue/server-renderer` 不支持此方法,它与 Node.js 环境分离。使用 [`pipeToNodeWritable`](#pipetonodewritable) 代替。
```
Comment (Jinjiang) [inline src/api/ssr.md:71]:
```suggestion
  `vue/server-renderer` 的 ESM 构建不支持此方法，因为它是与 Node.js 环境分离的。请换为使用 [`pipeToNodeWritable`](#pipetonodewritable)。
```

File: src/api/ssr.md [hunk 7]
```diff
@@ -90,15 +90,15 @@ Render and pipe to an existing [Node.js Writable stream](https://nodejs.org/api/
 - **示例**
 
   ```js
-  // inside a Node.js http handler
+  // 内置 Node.js http 处理器
```
Comment (Jinjiang) [inline src/api/ssr.md:93]:
```suggestion
  // 在一个 Node.js http 处理函数内
```

File: src/api/ssr.md [hunk 8]
```diff
@@ -90,15 +90,15 @@ Render and pipe to an existing [Node.js Writable stream](https://nodejs.org/api/
...
   pipeToNodeWritable(app, {}, res)
   ```
 
 ## renderToWebStream()
 
-Renders input as a [Web ReadableStream](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API).
+渲染输入作为 [Web ReadableStream](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API)。
```
Comment (Jinjiang) [inline src/api/ssr.md:99]:
```suggestion
[Web ReadableStream](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API) 形式的渲染输入。
```

File: src/api/ssr.md [hunk 9]
```diff
@@ -112,19 +112,19 @@ Renders input as a [Web ReadableStream](https://developer.mozilla.org/en-US/docs
 - **示例**
 
   ```js
-  // inside an environment with ReadableStream support
+  // 内置一个支持ReadableStream的环境
```
Comment (Jinjiang) [inline src/api/ssr.md:115]:
```suggestion
  // 在一个支持 ReadableStream 的环境下
```

File: src/api/ssr.md [hunk 10]
```diff
@@ -112,19 +112,19 @@ Renders input as a [Web ReadableStream](https://developer.mozilla.org/en-US/docs
...
   return new Response(renderToWebStream(app))
   ```
 
-  :::tip Note
-  In environments that do not expose `ReadableStream` constructor in the global scope, [`pipeToWebWritable()`](#pipetowebwritable) should be used instead.
+  :::tip 备注
+  在这些环境中，不能在全局作用域暴露出 `ReadableStream` 的构造函数，使用 [`pipeToWebWritable()`](#pipetowebwritable) 代替。
```
Comment (Jinjiang) [inline src/api/ssr.md:120]:
```suggestion
  在不能全局暴露 `ReadableStream` 构造函数的环境下，请换为使用 [`pipeToWebWritable()`](#pipetowebwritable)。
```

File: src/api/ssr.md [hunk 11]
```diff
@@ -138,11 +138,11 @@ Render and pipe to an existing [Web WritableStream](https://developer.mozilla.or
 
 - **示例**
 
-  This is typically used in combination with [`TransformStream`](https://developer.mozilla.org/en-US/docs/Web/API/TransformStream):
+  通常与 [`TransformStream`](https://developer.mozilla.org/en-US/docs/Web/API/TransformStream) 结合使用：
 
   ```js
-  // TransformStream is available in environments such as CloudFlare workers.
-  // in Node.js, TransformStream needs to be explicitly imported from 'stream/web'
+  // 诸如CloudFlare workers这样的环境中，TransformStream 是可用的。
```
Comment (Jinjiang) [inline src/api/ssr.md:144]:
```suggestion
  // 诸如 CloudFlare worker 这样的环境中，TransformStream 是可用的。
```

File: src/api/ssr.md [hunk 12]
```diff
@@ -138,11 +138,11 @@ Render and pipe to an existing [Web WritableStream](https://developer.mozilla.or
...
+  // 在 Node.js， TransformStream 需要从 'stream/web' 显示导入
```
Comment (Jinjiang) [inline src/api/ssr.md:145]:
```suggestion
  // 在 Node.js 中，TransformStream 需要从 'stream/web' 显示导入。
```

File: src/api/ssr.md [hunk 13]
```diff
@@ -196,7 +196,7 @@ Renders input in streaming mode using a simple readable interface.
 
 ## useSSRContext()
 
-A runtime API used to retrieve the context object passed to `renderToString()` or other server render APIs.
+一种用于检索已传递给 `renderToString()` 或其他服务端渲染API上下文对象的运行时API。
```
Comment (Jinjiang) [inline src/api/ssr.md:199]:
```suggestion
一个运行时 API，用于获取已传递给 `renderToString()` 或其他服务端渲染 API 的上下文对象。
```

File: src/api/ssr.md [hunk 14]
```diff
@@ -206,17 +206,17 @@ A runtime API used to retrieve the context object passed to `renderToString()` o
 
 - **示例**
 
-  The retrieved context can be used to attach information that is needed for rendering the final HTML (e.g. head metadata).
+  被检索的上下文能够作为附加信息用于渲染最终的 HTML （例如头部元数据）
```
Comment (Jinjiang) [inline src/api/ssr.md:209]:
```suggestion
  得到的上下文能够作为附加信息用于渲染最终的 HTML (例如 head 中的元数据)。
```

**Pull request merged**